docs/references/http-websocket-apis/public-api-methods/path-and-order-book-methods/mpt_holders.md

```diff
@@ -0,0 +1,164 @@
+| `marker`          | string               | No        | Used to continue your query where it left off in paginating. |
+| `limit`           | number (positive integer) | No   | Specify a limit to the number of MPTs returned. |
+
+## Response Format
+
+```json
+{
+    "mpt_issuance_id": "000004C463C52827307480341125DA0577DEFC38405B0E3E",
+    "limit":50,
+    "ledger_index": 2,
+    "mptokens": [{
+        "account": "rEiNkzogdHEzUxPfsri5XSMqtXUixf2Yx",
+        "flags": 0,
+        "mpt_amount": "20",
+        "locked_amount": "1",
+        "mptoken_index": "36D91DEE5EFE4A93119A8B84C944A528F2B444329F3846E49FE921040DE17E65"
+    },
+    {
+        "account": "rrnAZCqMahreZrKMcZU3t2DZ6yUndT4ubN",
+        "flags": 0,
+        "mpt_amount": "1",
+        "mptoken_index": "D137F2E5A5767A06CB7A8F060ADE442A30CFF95028E1AF4B8767E3A56877205A"
+    }],
+    "validated": true
+}
+```
+
+### Response Fields
+
+The response follows the [standard format](https://xrpl.org/docs/references/http-websocket-apis/api-conventions/response-formatting/), with the result containing the following fields:
+
+| Field                  | Type    | Description                               |
+|:-----------------------|:--------|:------------------------------------------|
+| `mpt_issuance_id`      | string  | The `MPTokenIssuance` queried             |
+| `mptokens`             | array   | An array of mptokens. Includes all relevant fields in the underlying MPToken object. |
+| `marker`               | string  | Used to continue querying where we left off when paginating. Omitted if there are no more entries after this result. |
+| `limit`                | number  | The limit, as specfied in the request
+| `ledger_index`         | number  | The index of the ledger used. |
+
+An `mptoken` object has the following parameters:
+
+| Field                  | Type    | Description |
+|:-----------------------|:--------|:------------------------------------------|
+| `account`              | string  | The account address of the holder who owns the `MPToken`. |
+| `flags`                | number  | The flags assigned to the`MPToken` object. |
+| `mpt_amount`           | string  | Base 10-encoded amount of the holder's balance. |
+| `locked_amount`        | string  | Base 10-encoded amount of the locked balance. (Can be omitted if the value is 0.) |
+| `mptoken_index`        | string  | Key of the `MPToken` object. |
+
+#### Synthetic mpt_issuance_id field
+`MPTokenIssuanceID` is an identifier that allows you to specify an `MPTokenIssuance` in RPCs. The server adds a synthetically parsed `mpt_issuance_id` field to API responses to avoid the need for client-side parsing of the `MPTokenIssuanceID`.
+
+##### Transaction Metadata
+An `mpt_issuance_id` field is provided in JSON transaction metadata (not available for binary) for all successful `MPTokenIssuanceCreate` transactions. The following APIs are impacted: `tx`, `account_tx`, `subscribe` and `ledger`.
+
```

Comment:
rest of the file addition doesn't feel belong in this file. [This](https://xrpl.org/docs/references/protocol/transactions/metadata) is a better place for them